.github/workflows/e2e.v2.yaml

```
@@ -0,0 +1,192 @@
+        # chaos: [null, failure, kill, partition, bandwidth]
+        include:
+          - mode: stream
+            deployment: helm-chart
+            environment: profile
+        exclude:
+          - deployment: helm-operator
+            environment: management
+          - deployment: helm-operator
+            environment: mirror
+          - deployment: helm-operator
+            environment: readreplica
```

actionlint (1.7.4)

value "readreplica" in "exclude" does not match in matrix "environment" combinations. possible values are "null", "management", "mirror", "profile"

(matrix)